Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -222,8 +222,7 @@ public class PBXProjGenerator {
         let projectAttributes: [String: Any] = ["LastUpgradeCheck": project.xcodeVersion]
             .merged(project.attributes)
 
-        let knownRegions = sourceGenerator.knownRegions.sorted()
-        pbxProject.knownRegions = knownRegions.isEmpty ? ["en"] : knownRegions
+        pbxProject.knownRegions = sourceGenerator.knownRegions.union(["Base"]).sorted()
```

Comment:
It looks like if a project has no localized files it will now default to having a `Base` instead of `en`. I guess that's ok. I can't imagine it breaking anything?